mailpoet/lib/Entities/SegmentEntity.php

```diff
@@ -68,8 +68,8 @@ class SegmentEntity {
   private $averageEngagementScoreUpdatedAt;
 
   /**
-   * @ORM\Column(type="integer", nullable=true, options={"default" : 0})
-   * @var int|null
+   * @ORM\Column(type="boolean", nullable=true)
+   * @var bool|null
```

Comment:
Is there a reason why we need the column to be nullable?
In migration, we set the default to 0, so during the alter command, all rows will be set to 0.

I think we could get rid of nullability. Then we could set default directly to the property.
```php
private $displayInManageSubscriptionPage = false;
```
and also the getter `getDisplayInManageSubscriptionPage` would always return a boolean.

What do you think?